icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -1,4 +1,4 @@
-// © 2016 and later: Unicode, Inc. and others.
```

Comment:
this is a mistake. I somehow mistakenly remove this line. Will add them back one I solve my h/w issue